### ISSUE TITLE
Random markdown output

### ISSUE THREAD

**Issue by Jab2870** [tldr-pages/tldr-node-client#240]
## Expected behaviour
When running `tldr --markdown --random` or `tldr --random --markdown`, it should output a random page in markdown format

## Actual behaviour
It outputs a random page in normal format

## Log, debug output
```
tldr --markdown --random
PAGE gcc

  gcc

  Preprocesses and compiles C and C++ source files, then assembles and links them together.

  - Compile multiple source files into executable:
    gcc source1.c source2.c -o executable

  - Allow warnings, debug symbols in output:
    gcc source.c -Wall -Og -o executable

  - Include libraries from a different path:
    gcc source.c -o executable -Iheader_path -Llibrary_path -llibrary_name

  - Compile source code into Assembler instructions:
    gcc -S source.c

  - Compile source code without linking:
    gcc -c source.c
```

## Environment
- Operating system - ~~OSX~~/**Linux**/~~Windows (add version if needed)~~
- NodeJS version  **11.4.0**

**Comment by Jab2870** [comment]
I have this line in my .{shell}rc files:

 ```
tldr --linux -r
```
Which means I get a random TLDR when I open a new terminal.

I also have this in my i3 config:

```
bindsym $mod+Shift+m exec "/usr/bin/tldr --list-all | sed 's/, /\\n/g' | rofi -dmenu -p 'TL;DR' -matching regex | xargs tldr -m | pandoc -f markdown -t ms | groff -ms - -T pdf | zathura -"
```

This lets me search the TLDR entries and opens a nicely formatted PDF of the entry chosen.

I wanted to merge the two and have a random one open as a PDF on a keyboard shortcut. (People tell me I have too much time haha)
